src/aspire/volume/__init__.py

```diff
@@ -323,6 +449,88 @@ class FBBasisVolume(BasisVolume):
     pass
 
 
+def _gaussian_blob_Cn_vols(
+    L=8, C=2, K=16, alpha=1, order=1, seed=None, dtype=np.float64
+):
+    """
+    Generate Cn rotationally symmetric volumes composed of Gaussian blobs.
+    The volumes are symmetric about the z-axis.
+
+    Defaults to volumes with no symmetry.
+
+    :param L: The size of the volumes
+    :param C: The number of volumes to generate
+    :param K: The number of blobs each volume is composed of.
+    A Cn symmetric volume will be composed of n times K blobs.
+    :param order: The order of cyclic symmetry.
+    :param alpha: A scale factor of the blob widths
+
+    :return: A Volume instance containing C Gaussian blob volumes with Cn symmetry.
+    """
+
+    # Evaluate the symmetrized gaussian blobs.
+    def _eval_gaussians(L, K, Q, D, mu, dtype=np.float64):
```

Comment:
This and `_gen_gaussians` do not depend on the symmetry type so they can be moved outside.

Reply:
Done.